homeassistant/components/xiaomi_miio/switch.py

```diff
@@ -120,6 +140,52 @@
         "method": "async_set_power_price",
         "schema": SERVICE_SCHEMA_POWER_PRICE,
     },
+    SERVICE_SET_BUZZER: {
+        "method_on": "async_set_buzzer_on",
+        "method_off": "async_set_buzzer_off",
+    },
+    SERVICE_SET_CHILD_LOCK: {
+        "method_on": "async_set_child_lock_on",
+        "method_off": "async_set_child_lock_off",
+    },
+    SERVICE_SET_DRY: {
+        "method_on": "async_set_dry_on",
+        "method_off": "async_set_dry_off",
+    },
+}
+
+
+@dataclass
+class SwitchType:
+    """Class that holds device specific info for a xiaomi aqara or humidifiers."""
+
+    name: str = None
+    short_name: str = None
+    unit_of_measurement: str = None
+    icon: str = None
+    device_class: str = None
+    service: str = None
+
+
+SWITCH_TYPES = {
+    FEATURE_SET_BUZZER: SwitchType(
+        name="Buzzer",
+        icon="mdi:volume-high",
+        short_name=ATTR_BUZZER,
+        service=SERVICE_SET_BUZZER,
+    ),
+    FEATURE_SET_CHILD_LOCK: SwitchType(
+        name="Child lock",
+        icon="mdi:lock",
+        short_name=ATTR_CHILD_LOCK,
+        service=SERVICE_SET_CHILD_LOCK,
+    ),
+    FEATURE_SET_DRY: SwitchType(
+        name="Dry mode",
```

Comment:
```suggestion
        name="Dry Mode",
```

Reply:
Updated in https://github.com/home-assistant/core/pull/52172